### ISSUE TITLE
Modify User Flow Routes: step 2

### ISSUE THREAD

**Issue by wimo7083** [OperationCode/operationcode_backend#472]
# Feature
## Why is this feature being added?
Updates the backend flow to provide an easier experience for users

## What should your feature do?
1. Flesh out requirements for this with @kylemh 
2. Email magic link to provide passwordless accounts
3. Create user on initial email/ password with reminder email when user doesn't finish.

**Comment by kylemh** [comment]
Blocked by the need to discuss further. We have tons of ideas that we can take to improve registration. We should discuss with stakeholders for what sounds best.

**Comment by wimo7083** [comment]
No longer needed.
